CorePluginFramework/src/au/gov/asd/tac/constellation/plugins/parameters/types/PasswordParameterType.java

```diff
@@ -78,7 +78,7 @@ public PasswordParameterType() {
 
     @Override
     public String validateString(PluginParameter<PasswordParameterValue> param, String stringValue) {
-        if (StringUtils.isNotBlank(stringValue) && stringValue.trim().isEmpty()) {
+        if (StringUtils.isBlank(stringValue)) {
```

Comment:
Hmm the problem with this change means that it is not valid to have no password. I don't see an issue with this but just in case there was an issue where `null` was valid, can you please update `CHANGELOG.md` to state the change in behaviour @Atlas139mkm ?

Reply:
If we want to leave it in I could change isNotBlank to IsBlank? The issue at the moment is its contradicting statement.
(if the string is not blank and the string is empty) - this means no string will ever pass through this method

Reply:
@arcturus2 if you look at the contents of the resulting code block, it simply returns "Parameter is empty" which was something that was already there prior to this PR. That seems to imply it was never the intention to allow no password to be valid and as @Atlas139mkm rightfully points out, the current boolean statement can never be true since a string can never be both blank and not blank at the same time. In this case, I'm probably ok with no changlog entry since this seems to be fixing behaviour rather than changing it

Reply:
Without trying to derail the PR review, does the database importer use the password parameter type for the credentials login? I believe @Auriga2 worked on this recently, can you shed some light on the above? 
Just wondering if this will disallow the entry of an empty password..

Reply:
Thanks @aldebaran30701.  I was actually thinking about this side affect but couldn't figure out something to test it against. I'll test the DB importer on this branch as I've been doing some testing on it recently I have an example I can use.

Reply:
Tested it @aldebaran30701  and it works OK.